tests/integration/test_api_resources.py

```diff
@@ -1,379 +1,320 @@
 
 from .utils import create_file_group, skip_on_travis, upload_tmp_txt_file
 
+
 # increase throttle retries for Travis CI
 conf.retry_throttled = 10
 
+IMAGE_URL = (
+    "https://github.githubassets.com/images/modules/logos_page/Octocat.png"
+)
 
-class RestAPITestCase(unittest.TestCase):
-    image_url = 'https://github.githubassets.com/images/modules/logos_page/Octocat.png'
 
-    @classmethod
-    def setUpClass(cls):
-        conf.pub_key = 'demopublickey'
-        conf.secret = 'demosecretkey'
+@pytest.fixture
+def group():
+    group = create_file_group(files_qty=1)
+    yield group
+    for file in group:
+        file.delete()
 
-    @classmethod
-    def tearDownClass(cls):
-        conf.pub_key = None
-        conf.secret = None
 
+def test_successful_upload_when_file_is_opened_in_txt_mode(small_file):
+    with open(small_file.name, "rt") as fh:
+        file = File.upload(fh)
 
-class FileUploadTest(RestAPITestCase):
+    assert isinstance(file, File)
 
-    def setUp(self):
-        self.tmp_txt_file = NamedTemporaryFile(mode='wb', delete=False)
-        self.tmp_txt_file.write('привет'.encode('utf-8'))
-        self.tmp_txt_file.close()
 
-    def test_successful_upload_when_file_is_opened_in_txt_mode(self):
-        with open(self.tmp_txt_file.name, 'rt') as fh:
-            file_ = File.upload(fh)
+def test_successful_upload_when_file_is_opened_in_binary_mode(small_file):
+    with open(small_file.name, "rb") as fh:
+        file = File.upload(fh)
 
-        self.assertIsInstance(file_, File)
+    assert isinstance(file, File)
 
-    def test_successful_upload_when_file_is_opened_in_binary_mode(self):
-        with open(self.tmp_txt_file.name, 'rb') as fh:
-            file_ = File.upload(fh)
 
-        self.assertIsInstance(file_, File)
+def test_get_some_token():
+    file_from_url = File.upload_from_url(IMAGE_URL)
+    assert file_from_url.token
 
 
-class FileUploadFromUrlTest(RestAPITestCase):
-    def test_get_some_token(self):
-        file_from_url = File.upload_from_url(self.image_url)
-        self.assertTrue(file_from_url.token)
+@skip_on_travis
+def test_successful_upload_from_url():
+    file_from_url = File.upload_from_url(IMAGE_URL)
 
-    @skip_on_travis
-    def test_successful_upload_from_url(self):
-        file_from_url = File.upload_from_url(self.image_url)
+    timeout = 30
+    time_started = time.time()
 
-        timeout = 30
-        time_started = time.time()
+    while time.time() - time_started < timeout:
+        status = file_from_url.update_info()["status"]
+        if status in ("success", "failed", "error"):
+            break
+        time.sleep(1)
 
-        while time.time() - time_started < timeout:
-            status = file_from_url.update_info()['status']
-            if status in ('success', 'failed', 'error'):
-                break
-            time.sleep(1)
+    assert isinstance(file_from_url.get_file(), File)
+
+
+@skip_on_travis
```

Comment:
rename to `skip_on_ci`

Reply:
didn't notice any problems with tests on CI, removed